### DISCUSSION TITLE
Table panel footer - footer should support different funtions for different columns

### DISCUSSION THREAD

**Discussion by tcgjensen** [grafana/grafana#63724] [Dashboards, Panels & Visualizations]
It would be great if it was possible to specify per column which function you want to use. Currently it is only possible to select one which is then used on the columns you specify.
I want some columns to be averaged, some summarized etc...

<img width="1248" alt="image" src="https://user-images.githubusercontent.com/26411913/221184979-a3edb6a3-a790-422e-b704-446a1a1aa76f.png">

On the picture below, you can see I can select one function and then select which columns to use it on, but that is not enough with just one function....
<img width="390" alt="image" src="https://user-images.githubusercontent.com/26411913/221185531-24f1f42c-79e9-4503-ae33-187c876c0dcf.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82841. Feel free to continue the discussion around this there. Thank you!
